app/controllers/applications_controller.rb

```diff
@@ -107,6 +107,23 @@ def show
     # Required for new email alert signup form
     @alert = Alert.new(address: @application.address)
 
+    if current_user && current_user.admin?
+      if params[:with_councillors] == "true"
+        @application_authority_councillors = [{ id: "1", name: "Mark Gardiner", party: "Independent" },
```

Comment:
Instead of manually putting an ID on each of these you could use `#each_with_index`. The order of an Array is predictable.